docs/admin-guide.md

```diff
@@ -313,6 +314,28 @@ pipelines:
     targets: *generic_targets # using YAML Alias
 ```
 
+#### Deploying to an OU target
+
+If you want to target all the AWS accounts belonging to an Organization Unit, you can pass the OU ID in the target path attribute, like in the below example. 
+
+This piece of functionality, coupled with a CloudFormation deployment target is equivalent to a CloudFormation StackSet deployed to all the accounts of the OU, with the benefit that it automatically handles new accounts added to the OU, which isn't currently supported by CloudFormation StackSets.
+
+```yaml
+  - name: example
+    default_providers:
+     source:
+        provider: codecommit
+        properties:
+          account_id: 11111111111111 # <-- This AWS account that hosts the source of the CloudFormation payload deployed by this pipeline.
+    targets:
+      deploy:
+        provider: cloudformation
+    targets:
+      - path: ou-d34d-b33f # <-- This is the OU ID you can get from the Organizations section of the AWS console
```

Comment:
Thanks for the PR! This section could also detail targeting an OU with tis path (ie. /marketing/production) which does the same thing as using the OU id. We should attempt to merge all information about targeting OU's for deployments into one documentation section :)

Reply:
Thanks, I was not aware of this alternative way, I'll edit the PR shortly to mention it as well. Or maybe even instead of the ID, what do you think?

Reply:
Hi @cristim, thanks for your contribution. Yes, that would have my preference too, referencing the OU path. OU-ids are hardly readable.
Currently, your changes are applied in the `admin-guide.md`, but I think the `user-guide.md` would make most sense to add these. I had to get used to this difference in the beginning too.
Maybe it would be good to highlight the difference in the docs at the top, pointing to the other doc.